process/PC_PL_Template_Markdown.md

```diff
@@ -113,7 +125,7 @@
 ### Feature Requirements
 *Features are more granular than Components.* 
 *For SW porting projects, this list serves as the detailed project reference for features*
-*For IP Cores or more complext projects, a user manual with requirements specification is produced at the PA gate, which may supercede this list of features*
+*For IP Cores or more complext projects, a user manual with requirements specification is produced at the PA gate, which may supersede this list of features*
```

Comment:
Just to check, I presume we are using US spelling throughout. "Supercede" would be British English spelling